sdk/identity/azure-identity/samples/azure-aad-auth-with-redis-py.md

```diff
@@ -84,17 +100,34 @@ Integrate the logic in your application code to fetch a Microsoft Entra access t
 import time
 import logging
 import redis
+import base64
+import json
 from azure.identity import DefaultAzureCredential
 
 scope = "https://redis.azure.com/.default"  # The current scope is for public preview and may change for GA release.
 host = ""  # Required
 port = 6380  # Required
-user_name = ""  # Required
+
+def extract_username_from_token(token):
```

Comment:
Would it be better to use jwt library in python here ?

```python
def extract_username_from_token(token):
    # Decode the JWT without verifying the signature (verify=False)
    decoded_jwt = jwt.decode(token, options={"verify_signature": False})
    
    # Extract the 'oid' field
    return decoded_jwt.get('oid')
    ```

Reply:
I am leaning towards no. :)

The latest release of jwt library was in 2021 https://pypi.org/project/jwt/1.3.1/.

We don't want to recommend our customers to use it by adding it in our sample code.